scripts/chi-classnames/replacement.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+pathtofolder='path/to/project/folder'
```

Comment:
Please, use uppercase for constants. I.e PROJECT_SRC or SRC or PATH_TO_PROJECT_FILES...